src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx

```diff
@@ -48,28 +51,44 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
 
   return (
     <div className="mt-4 flex flex-col items-center justify-center text-center">
-      <Image src="/assets/images/dancing.gif" width={218} height={218} />
-      <div className="pt-8 text-4xl font-medium">Congratulations!</div>
-      <div className="mt-6 text-xl font-normal">{deployGreeting}</div>
-      <div className="flex flex-col gap-2 py-12">
-        <Button
-          type="primary"
-          onClick={handleGoToProject}
-          loading={gotoProjectClicked}
-        >
-          <span>
-            <Trans>Go to project</Trans>
-          </span>
-        </Button>
+      <Image src={DeploySuccessHero} width={380} height={379} />
+      <div className="pt-8 font-display text-5xl font-bold">
+        <Trans>Congratulations!</Trans>
+      </div>
+      <div className="mt-4 text-lg font-normal text-grey-600 dark:text-slate-200">
+        {deployGreeting}
+      </div>
+      <XLButton
+        type="primary"
+        onClick={handleGoToProject}
+        loading={gotoProjectClicked}
+        className="mt-8 flex items-center gap-3 text-white"
+      >
+        <span className="text-[16px]">
```

Comment:
never need to use px in font sizes. `text-base` will work here

Reply:
Ah true that's right, forgot about `base`